lib/chef/resource/mount.rb

```diff
@@ -65,7 +66,7 @@ class Mount < Chef::Resource
 
       property :options, [Array, String, nil],
         description: "An array or comma separated list of options for the mount.",
-        coerce: proc { |arg| arg.is_a?(String) ? arg.split(",") : arg },
+        coerce: proc { |arg| mount_options(arg) }, # Please see #mount_options method.
```

Comment:
Any time we do a coerce we should make sure to add specs to test the various scenarios

Reply:
I added those here so we don't block on merging this: https://github.com/chef/chef/pull/10625

Reply:
> Any time we do a coerce we should make sure to add specs to test the various scenarios

@tas50 Okay, I'll do that.